### PR TITLE
Create some meta templates

### LINKED ISSUE #18: Addition of meta templates
I believe that a template system for the issues on this repository would aid in the lowering of time required to create an issue and, moreover, mean that the required detail is present in more cases.

There could be multiple of these templates, for example one for 'Feature Requests'.

### PR DESCRIPTION
**Pull request by ZomBMage**
Create ISSUE_TEMPLATE.md with sample issue template.

Resolves #18

### CODE REVIEW

**Comment by NicholasG04** [comment]
Like it, could we get one for a feature request implemented too? Replacing issue with a request?

**Comment by ZomBMage** [comment]
I can make another template later for that, once I am home tonight. Thanks for the suggestion

**Comment by teamshortcut** [comment]
Looks good, feature request would also be useful. Will wait until added for review.

**Comment by ZomBMage** [comment]
Feature/Suggestion Request template added.

File: FEATUREREQ_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Title Of Feature/Suggestion
+
+### *A description of the feature/suggestion:*
+[fill out]
+
+
+
+### *Optional: Reasoning / Explaination:*
```
Comment (teamshortcut) [inline FEATUREREQ_TEMPLATE.md]:
```suggestion
### *Reasoning / Explaination:*
```
Minor edit, otherwise looks good.

**Review by teamshortcut — CHANGES_REQUESTED** [review]

**Review by teamshortcut — APPROVED** [review]

**Comment by thebeanogamer** [comment]
:eyes:
Screenshot_20190514-180158

**Review by NicholasG04 — CHANGES_REQUESTED** [review]
'Explanation' spelt incorrectly.

**Comment by ZomBMage** [comment]
That's FEATURE*REQ*, short for Feature Request. Didn't want the filename too long!

**Review by NicholasG04 — APPROVED** [review]
Thanks :)

**Comment by NicholasG04** [comment]
@teamshortcut @thebeanogamer can we get this merged and set up?

**Review by thebeanogamer — APPROVED** [review]

**Pull request merged**